src/api/txMining.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import txMiningRequestClient from './txMiningAxios';
+
+/**
+ * Api calls for tx mining
+ *
+ * @namespace ApiTxMining
+ */
+
+const txMiningApi = {
+  /**
+   * Submit tx to be mined
+   *
+   * @param {String} tx data in hexadecimal
```

Comment:
Missing other parameters

Reply:
I will add all docstrings soon and then remove from draft

Reply:
Ah, my bad, didn't see that on your original post.